pkg/workloadmeta/types.go

```diff
@@ -531,6 +532,7 @@ func (p KubernetesPod) String(verbose bool) string {
 		_, _ = fmt.Fprintln(&sb, "PVCs:", sliceToString(p.PersistentVolumeClaimNames))
 		_, _ = fmt.Fprintln(&sb, "Kube Services:", sliceToString(p.KubeServices))
 		_, _ = fmt.Fprintln(&sb, "Namespace Labels:", mapToString(p.NamespaceLabels))
+		_, _ = fmt.Fprintln(&sb, "Finished At:", p.FinishedAt)
```

Comment:
Should we print this only if it's non-zero?

Reply:
I think that makes sense. I'll make the change!